### PR TITLE
add post link and captcha api

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses: @ajacksified or @danehansen

### CODE REVIEW

File: src/endpoints/v1.es6.js [hunk 1]
```diff
@@ -461,6 +461,54 @@ class APIv1Endpoint {
...
+            options.form.text = json.text;
+          } else if (json.url) {
+            options.form.url = json.url;
+          }
+
+          return basePost(uri, options, this.request, (body) => {
+            if (body.json && body.json.errors.length < 1) {
+              // TODO handle errors
+              if (!body.json.errors.length) {
+                return body.json.data
```
Comment (ajacksified) [inline src/endpoints/v1.es6.js]: `;`

File: src/endpoints/v1.es6.js [hunk 2]
```diff
@@ -461,6 +461,54 @@ class APIv1Endpoint {
...
+              }
+              // var link = body.json.data.things[0].data;
```
Comment (ajacksified) [inline src/endpoints/v1.es6.js]: nuke

File: src/endpoints/v1.es6.js [hunk 3]
```diff
@@ -461,6 +461,54 @@ class APIv1Endpoint {
...
+            } else {
+              return body.json.errors;
```
Comment (ajacksified) [inline src/endpoints/v1.es6.js]: might want to `defer.reject` out the errors instead of returning them

File: src/endpoints/v1.es6.js [hunk 4]
```diff
@@ -693,6 +741,34 @@ class APIv1Endpoint {
     }, this)
   }
 
+  get captcha () {
+    return bind({
+      get: function (options = {}) {
```
Comment (curioussavage) [inline src/endpoints/v1.es6.js]: forgot I removed use of this endpoint. I suppose I should remove this

File: src/endpoints/v1.es6.js (hunk 3 above)
Comment (curioussavage) [inline src/endpoints/v1.es6.js]: When you have a moment could you show me how to do this? It's not calling the reject function.

File: src/endpoints/v1.es6.js (hunk 4 above)
Comment (ajacksified) [inline src/endpoints/v1.es6.js]: If it's functional, might as well leave it in here.

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**